library/std/src/os/unix/net/ancillary.rs

```diff
@@ -249,46 +249,46 @@ impl SocketCred {
     #[unstable(feature = "unix_socket_ancillary_data", issue = "76915")]
     #[must_use]
     pub fn new() -> SocketCred {
-        SocketCred(libc::cmsgcred { cmsgcred_pid: 0, cmsgcred_uid: 0, cmsgcred_gid: 0 })
+        SocketCred(libc::cmsgcred { cmcred_pid: 0, cmcred_uid: 0, cmcred_gid: 0 })
```

Comment:
This still appears wrong to me. DragonFly defines additional members `cmsgcred` which aren't being referenced here, leading to [E0063](https://doc.rust-lang.org/error-index.html#E0063) when I build this locally. https://github.com/rust-lang/libc/blob/9b054d8779d2f71f1803e20f0e3b3327a4ae2764/src/unix/bsd/freebsdlike/mod.rs#L214-L221

@devnexen are you building these changes locally? My understanding is with DragonFly being a Tier 3 target, this code isn't actually built or tested in PR or CI builds, meaning you can't rely on bors to validate your changes.

I can walk you through building this locally if needed, but I'm concerned that neither this PR nor https://github.com/rust-lang/rust/commit/e68887e67cc6b7bb4ea5113a40eaa4c0831bda13 were ever actually tested...

Reply:
is that the only build error ?

Reply:
It is the only error which is not originating as a warning.

Are you set up to build this locally to test your changes?

Reply:
is it building for you ? unfortunately I lost my DragonFly VM so I can t test for now.

Reply:
sorry for the disturbance, for now better to revert at least that will unblock your situation.